__tests__/app.test.js

```diff
@@ -60,7 +61,8 @@ describe('app', () => {
                         review_body: expect.any(String),
                         category: expect.any(String),
                         created_at: expect.any(String),
-                        votes: expect.any(Number)
+                        votes: expect.any(Number),
+                        comment_count: expect.any(Number)
```

Comment:
It could be good to add an extra test here to check the comment count works. We can check one object specifically, such as [0] and check the comment_count is the number we would expect it to be. This is optional but could be good to add.